test/helpers/builders.js

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: strong-remoting
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+exports.remotingDate = function(val) {
+  return {
+    // NOTE(bajtos) The current implementation of strong-remoting
+    // calls `.toString()` which returns a date value like this:
+    //     Sat Jan 01 2000 00:00:00 GMT+0100 (CET)
+    // I think we should use `.toJSON()` instead, which returns concise
+    //     1999-12-31T23:00:00.000Z
+    '$data': new Date(val).toString(),
```

Comment:
Let's discuss. I can see two approaches: 

1. A global feature flag that will control whether date is serialized via `toString()` or `toJSON()`. The application author decides the response format for all clients.

2. A special request argument (or HTTP header) allowing the client to request dates in the old strong-remoting@2.x format.

Thoughts?